hack/check-gh-chart-versions.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+TEXT_RED='\033[0;31m'
+TEXT_RESET='\033[0m'
+TEXT_GREEN='\033[0;32m'
+
+target_version=$1
+if [[ $# -eq 0 ]]; then
+    echo "Release version argument is required"
+    echo
+    echo "Usage:  ${0} <release version>"
+    exit 1
+fi
+
+chart_dir=$(pwd)/charts
+
+controller_version=$(cat $chart_dir/gha-runner-scale-set-controller/Chart.yaml | yq .version)
```

Comment:
```suggestion
controller_version="$(cat $chart_dir/gha-runner-scale-set-controller/Chart.yaml | grep version: | cut -d ' ' -f 2)"
```

Let's not introduce `yq` into the mix

Reply:
yq is already a dependency https://github.com/actions/actions-runner-controller/blob/9859bbc7f2b1d9006fd137e6ed1db807cbfe86a0/Makefile#L344

And I think it would be nicer to use yq since yaml like this should return the same value:
```yaml
version: 0.1.0
appVerison: "0.1.0"
```
yq would evaluate correctly while grep will have a mismatch